remote/src/main/resources/reference.conf

```diff
@@ -198,6 +201,12 @@ pekko {
     # nodes have been are running on Apache Pekko
     protocol-name = "pekko"
 
+    # When pekko.remote.accept-protocol-names contains "akka", then we
+    # need to know the Akka version. If you include the Akka jars on the classpath,
+    # we can use the akka.version from their configuration. This configuration
+    # setting is only used if we can't find an akka.version setting.
+    akka.version = "2.6.21"
```

Comment:
This might be over-engineering way too much but considering you could put any string values into `accept-protocol-names` it might make sense for this configuration to be an object i.e.

```
version {
  akka = "2.6.21"
}
```
where the key (in this case `akka`) is pointing to the the same value inside of ` accept-protocol-names`.

Otherwise we should do some sanity validation on `accept-protocol-names` and refuse any values that aren't `akka` or `pekko`

Reply:
I think this is overengineering. The new code I added is very Akka specific (#765). It only kicks if you include "akka" in `accept-protocol-names` (again hardcoded). In the real world, I do not know of anyone else who would need to add a non-Pekko node to a Pekko cluster. If that story ever comes to pass, I will volunteer to help those users to get the cluster compatibility checks to work with their nodes.

Reply:
Can we then just add a validation check to the config parsing so that only `akka` and `pekko` can be put in as values (can also be separate PR)?